hangman.rb

```diff
@@ -0,0 +1,195 @@
+require 'colorize'
+
+class Hangman
+	def initialize
+		@words = ["method", "index", "bracket", "braces", "gem", "git", "terminal", "length", 
+							"count", "push", "class", "hash", "comment", "colorize", "mastermind", "orange", 
+							"online", "dragon", "seattle", "rubric"]
+		@word = @words.sample
+		@blank_spaces = "_ -" * @word.length
+		@blank_array = @blank_spaces.split("-")
+		@answer_letters = @word.split(//)
+		@wrong_answers = 0
+		@guess_array = []
+		@alphabet = %w(a b c d e f g h i j k l m n o p q r s t u v w x y z)
+		start
+	end
+
+	def start
+		puts """
+		Let's play Hangman!!!
+		Can you guess our Ada class related term?
+
+		|     _________
```

Comment:
Since this is a portion of the drawing, you could handle this condition in the `draw_hangman` method in order to encapsulate all of the drawing logic there rather than having some extraneous drawing in the start method.